src/words.ts

```diff
@@ -0,0 +1,127 @@
+type StepVector = [string, number][];
+
+export function add(v1: StepVector, v2: StepVector): StepVector {
+  const result = v1;
```

Comment:
This is very bad practice. This function should be called `addInPlace` and even then it would be unclear which vector is the output.

Reply:
Point taken. I'd rather ensure that `v1` and `v2` are never mutated. Cloning, I guess.